src/SourceBuild/content/eng/tools/tasks/PackageDiff/PackageDiff/PackageDiff.csproj

```
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
```

We don't want hardcoded TFMs as this makes management more difficult 
```suggestion
    <TargetFramework>$(NetCurrent)</TargetFramework>
```